GUI/add_new.py

```diff
@@ -55,8 +57,10 @@ def add_new_game(json_to_add, is_new):
 	json_list.append(json_to_add)
 	with open('temp_json', 'w', encoding="utf-8") as f:
 		 f.write(json.dumps(json_list, indent=4, sort_keys=True))
-	os.remove("games.json")
-	os.rename("temp_json", "games.json")
+		 ftp = ftplib.FTP("legtux.org", "dogeek", "uy7qa9e")
```

Comment:
Passwords shouldn't be stored in code (Also I hope you ftp server is locked down)